xla/service/buffer_assignment.cc

```diff
@@ -1941,8 +1954,8 @@ absl::Status BufferAssigner::AssignBuffersWithSequentialOrdering(
                   get_heap_algorithm(alignment), *private_stack_computation,
                   *instruction_sequence, assignment->alias_analysis(),
                   alias_info_, assignment->buffer_size_, &schedule, options));
-          AssignBuffersFromHeapSimulator(result, assignment, color,
-                                         isolation_options);
+          std::ignore = AssignBuffersFromHeapSimulator(
```

Comment:
Why do we ignore the returned status here? My guess is this would cause issues. At the very least, we should preserve the previous behavior and CHECK fail if the returned status here is not ok.